media/css/firefox/election.scss

```diff
@@ -18,6 +18,13 @@ $image-path: '/media/protocol/img';
     padding-bottom: 200px;
 }
 
+.mzp-c-hero-title {
+    @include at2x('/media/img/firefox/election/firefox-logo.png', 100px);
+    background-repeat: no-repeat;
+    background-position: top left;
```

Comment:
Should the default position for this be `top center`, to match the behavior of the default hero component?

![image](https://user-images.githubusercontent.com/400117/57027528-88c19d00-6c34-11e9-8c93-e1e0f7399924.png)

Reply:
✅